blog.md

```diff
@@ -1,53 +1,62 @@
-# Docker Voluminous plugin from ClusterHQ
+# Making the (bridging the gap/making the leap/easing the transition) development to production
```

Comment:
I like keeping this to just one line, ```#Easing the transition of development to production```, have (something/something/something) is hard to read.